tests/client/test_client_authorization_and_tenant.py

```
@@ -0,0 +1,110 @@
+    start = time.time()
+    host = "http://localhost:8080/v2/management"
+    client = ApiClient(
+        configuration=Configuration(host=host, api_key={"APIKeyAuth": "APIKey"}),
+        cookie="tenant=test-tenant",
+    )
+    api_api = ApiApi(client)
+    while time.time() - start < timeout:
+        try:
+            api_api.check_api_is_alive()
+            break
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)